cli/cli-client/src/main/resources/help/user.adoc

```diff
@@ -0,0 +1,98 @@
+USER(1)
+=======
+:doctype: manpage
+
+
+NAME
+----
+user - manages ${infinispan.brand.name} users in property security realms.
+
+
+SYNOPSIS
+--------
+*user ls*
+
+*user create* 'username'
+
+*user describe* 'username'
+
+*user remove* 'username'
+
+*user modify* 'username'
+
+
+DESCRIPTION
+-----------
+Manage (list, created, describe, remove, modify) users stored in a property security realm.
+Note: this command cannot interact with other security realms.
+
+
+COMMAND SYNOPSIS
+----------------
+
+List users in the property security realm
+
+*user ls*::
+Lists the users or groups which are present in the property files.
+
+Create a user
+
+*user create* 'username'::
+Creates a user. The command will prompt for a password and confirmation.
+
+Describes a user
+
+*user describe* 'username'::
+Describes a user, including its username, realm and any groups it belongs to.
+
+Remove a user
+
+*user remove* 'username'::
+Removes the specified user grom the property files.
+
+Modify a user
+
+*user modify* 'username'::
+Modifies a user by changing its password or the groups it belongs to.
+
+COMMON OPTIONS
```

Comment:
it needs 2 new lines between sections.
The same issue for `USER CREATE/MODIFY OPTIONS` section